src/aics-image-viewer/components/ErrorBanner/index.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { Alert, Button } from "antd";
+import { VolumeLoadError, VolumeLoadErrorType } from "@aics/volume-viewer";
+import { RightOutlined } from "@ant-design/icons";
+
+import "./styles.css";
+import { useConstructor } from "../../shared/utils/hooks";
+
+const IssueLink: React.FC<React.PropsWithChildren<{ bug?: boolean }>> = ({ bug, children }) => (
+  <a
+    href={`https://github.com/allen-cell-animated/website-3d-cell-viewer/issues/new${
+      bug ? "?template=bug_report.md" : "/choose"
+    }`}
+    target="_blank"
+    rel="noreferrer noopener"
+  >
+    {children}
+  </a>
+);
+
+const UNKNOWN_ERROR_DESCRIPTION: React.ReactNode = (
+  <>
+    An unknown error occurred. Check the browser console (F12) for more details. If this looks like a bug,{" "}
```

Comment:
help I'm bad at javascript: what does `{"  "}` do? Is that just forcing there to be a space even though it's the end of the line?

Reply:
Yeah, basically. It's a quirk of JSX (the syntax extension that lets you write React components like HTML `<></>` and have it be transformed into the right React function calls) that there's no other way to tell it there's a space between the text and the component on the next line. I think it's annoying, but the formatter wanted it there.